trojsten/reviews/tests.py

```diff
@@ -173,13 +173,9 @@ def test_redirect_to_login(self):
         # Najprv sa posle na login, potom na admin login, a az potom na povodnu stranku.
         # Posledna cast za next je double quoted, posledne next je len quoted.
         url = reverse(self.url_name, kwargs={"task_pk": 1})
-        response = self.client.get(url, follow=True)
-        login_url = settings.LOGIN_URL
-        admin_login_url = "?next=%s" % reverse("admin:login")
-        last_url = quote(url, safe="")
-        last_url = quote("?next=%s" % last_url, safe="")
-        redirect_to = "%s%s%s" % (login_url, admin_login_url, last_url)
-        self.assertRedirects(response, redirect_to)
+        response = self.client.get(url)
+        redirect_to = "%s?next=%s" % (reverse("admin:login"), url)
+        self.assertRedirects(response, redirect_to, fetch_redirect_response=False)
```

Comment:
Tieto testy mi nepridu ekvivalentne tym predoslym. Zmenilo sa nieco? Je komentar nad tym stale validny?

Reply:
Zmenilo sa to tak, ze povodne ta to presmerovalo na `/admin/login`, ktory ta hodil na `/ucet/login`, na to ale chces follow, ktory nemozes pouzit pri logine. Asi by sa hodilo zmenit ten komentar nad tym, no :)

Reply:
Updatnes pls tie komentare teda?